Tasks/Common/VstsAzureHelpers_/InitializeFunctions.ps1

```diff
@@ -174,12 +174,6 @@ function Initialize-AzureSubscription {
             # Else, this is AzureRM.
             
             try {
-                if(Get-Command -Name "Clear-AzureRmContext" -ErrorAction "SilentlyContinue"){
```

Comment:
We should not remove this. In private agents, users can still have AzureRM modules lesser than 6.7.0. We will start breaking them again.